tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -341,6 +341,7 @@ private static void GenerateCoverageFile(ApiMetadata api, string directory)
             var targetExecutable = new XElement("TargetExecutable", "/Program Files/dotnet/dotnet.exe");
             var targetArguments = new XElement("TargetArguments",
                 $"test --no-build -f {GetPreferredCoverageFramework(api)} -c Release");
+            var filters = new XElement("Filters", new XElement("IncludeFilters", new XElement("FilterEntry", new XElement("ModuleMask", api.Id))));
```

Comment:
Ah, just thought - removing the filter would bring back coverage of all the tests etc.
I'll file a bug for Spanner on this, and add it into apis.json for "extra assemblies to cover". (Or consider including all dependencies on APIs.)